blocks/init/src/Blocks/custom/quote/manifest.json

```diff
@@ -1,5 +1,6 @@
 {
 	"$schema": "https://raw.githubusercontent.com/infinum/eightshift-frontend-libs/develop/schemas/block.json",
+	"active": true,
```

Comment:
This is a change in the schema. The schema file should be updated as well to denote what this key does.